app/controllers/clearance/sessions_controller.rb

```diff
@@ -33,4 +40,8 @@ def url_after_create
   def url_after_destroy
     sign_in_url
   end
+
+  def url_for_signed_in_users
+    url_after_create
```

Comment:
There are several methods that point to `Clearance.configuration.redirect_url` and in this instance I think we actually want to redirect to the same place that users get redirected to after the session is created. This can always be overridden in the controller.